schema.graphql

```diff
@@ -17,44 +17,38 @@ type Protocol @entity {
     totalSettlementTokenBalance: BigDecimal!
     "total non settlement tokens"
     totalNonSettlementTokenBalances: [ProtocolTokenBalance!] @derivedFrom(field: "protocol")
-    "block number"
-    blockNumber: BigInt!
-    "block timestamp"
-    timestamp: BigInt!
```

Comment:
What's the alternative to these 2 fields? By query `indexingStatusForCurrentVersion` I guess?